### ISSUE TITLE
Entrega 3 Apartado X : Contraseña aleatoria derivada de nombre y horas gastadas

### ISSUE THREAD

**Issue by fernandolalanne** [IIC2413/Syllabus-2021-2#377]
> Ingresa los detalles de tu pregunta aquí, o da cualquier información  adicional al título que ayude al equipo docente a responder tu pregunta
Hola! Quería preguntar una duda del formato de la creación para las contraseñas aleatorias de los usuarios. Sucede que en el enunciado cuando se dice que "pero asignándoles una contraseña nueva derivada de su nombre y cantidad de horas gastadas jugando" se refiere a que si mi nombre es Fernando y llevo 15 horas jugando, ejemplos de contraseñas validas podrían ser

-Fernando15
-1Fernando5

Desde ya muchas gracias!

**Comment by arpincheira** [comment]
Efectivamente, podrían hacer que las contraseñas varíen de forma que se inserte cada uno de los números de las horas jugadas de un usuario en alguna parte de su nombre, otra opción podría ser mezclar dichos datos de forma aleatoria, por ejemplo:

Fernando 15 -> nen1darFo
